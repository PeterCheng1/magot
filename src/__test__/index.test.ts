import * as components from '..';

describe('component files', () => {
  it('should export components correctly', () => {
    expect(Object.keys(components)).toMatchSnapshot();
  });
});
