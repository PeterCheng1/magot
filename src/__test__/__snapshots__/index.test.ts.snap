// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component files should export components correctly 1`] = `
Array [
  "Animation",
  "AutoComplete",
  "Avatar",
  "Badge",
  "Breadcrumb",
  "Button",
  "ButtonGroup",
  "Calendar",
  "Card",
  "Carousel",
  "Checkbox",
  "CheckboxGroup",
  "Collapse",
  "ColorPicker",
  "ContextMenu",
  "DatePicker",
  "DateTimePicker",
  "Dialog",
  "Drawer",
  "Dropdown",
  "Empty",
  "FileButton",
  "Form",
  "Gallery",
  "Hotkey",
  "Icon",
  "Image",
  "Input",
  "InputNumber",
  "Layout",
  "List",
  "Loader",
  "Loading",
  "Menu",
  "Modal",
  "Notification",
  "Pagination",
  "Password",
  "Popover",
  "Popup",
  "Qrcode",
  "Radio",
  "RadioGroup",
  "Rate",
  "Search",
  "Select",
  "Slider",
  "Switch",
  "Table",
  "Tabs",
  "TextArea",
  "TimePicker",
  "Toast",
  "Tooltip",
  "Tree",
  "Uploader",
  "VideoPlayer",
]
`;
